src/main/java/fi/vm/yti/terminology/api/frontend/FrontendTermedService.java

```diff
@@ -250,6 +252,7 @@ GenericNodeInlined getConcept(UUID graphId, UUID conceptId) {
         if (result.size() == 0) {
             throw new NodeNotFoundException(graphId, conceptId);
         } else {
+            System.out.println("GetConcept " + result.get(0).getProperties().get("status").stream().iterator().next().getValue());
             return userNameToDisplayName(result.get(0), new UserIdToDisplayNameMapper(),
```

Comment:
Is this only for debugging purposes? Could use logger for this

Reply:
Seems like I still left some debugging code. Will remove that :+1: